<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>New_track</class>
 <widget class="QDialog" name="New_track">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.129353 rgba(75, 69, 224, 255), stop:0.502488 rgba(141, 85, 255, 255), stop:0.875622 rgba(130, 0, 255, 255));
font-family: Noto Sans SC;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255. 255. 40);
border-radius: 7px</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_new_track">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font-weight: bold;
font-size: 20pt;
background-color: none;
border: none;</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>Новая транзакция</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_category">
        <property name="styleSheet">
         <string notr="true">QComboBox {
font-size: 16pt;
color: white; 
}

QComboBox::item {
color: white;
}

QComboBox QAbstractItemView {
 color: white; 
 background-color: black; 
}</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>Покупки</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
        </property>
        <item>
         <property name="text">
          <string>Покупки</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Развлечения</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Одежда</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Другое</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Работа</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="data">
        <property name="styleSheet">
         <string notr="true">font-size: 16pt;
color: white;
padding-left: 10px</string>
        </property>
        <property name="date">
         <date>
          <year>2024</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_opisanie">
        <property name="styleSheet">
         <string notr="true">font-size: 16pt;
color: white;
padding-left: 10px</string>
        </property>
        <property name="placeholderText">
         <string>Подробности</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_summa">
        <property name="styleSheet">
         <string notr="true">font-size: 16pt;
color: white;
padding-left: 10px</string>
        </property>
        <property name="placeholderText">
         <string>Сумма </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <property name="styleSheet">
         <string notr="true">font-size: 16pt;
color: white;
padding-left: 10px</string>
        </property>
        <item>
         <property name="text">
          <string>Расход</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Доход</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_add">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font-weight: bold;
font-size: 15pt;
background-color: none;
border: 1px solid rgba(250, 250, 250, 40);
border-radius: 7px;
width: 230px;
height: 50px</string>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../Downloads/add_circle_24dp_FFFFFF_FILL0_wght400_GRAD0_opsz24.svg</normaloff>../../../Downloads/add_circle_24dp_FFFFFF_FILL0_wght400_GRAD0_opsz24.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
